docs/source/user/installation.rst

```diff
@@ -11,17 +11,15 @@ Supported GPUs
 --------------
 
 Numba supports all NVIDIA GPUs that are supported by the CUDA Toolkit it uses.
-Presently for CUDA 11 this ranges from Compute Capabilities 3.5 to 9.0, and for
-CUDA 12 this ranges from 5.0 to 12.1, depending on the exact installed version.
+Presently for CUDA 12 this ranges from Compute Capabilities 5.0 to 12.1, and for
+CUDA 13 this ranges from 5.0 to XXX, depending on the exact installed version.
```

Comment:
Do we want to leave this as `XXX` or update it as we go?

Reply:
We definitely need the lower bound to be 7.5. Maxwell, Pascal, and Volta were dropped in CUDA 13. I'd leave the upper bound at 12.1 but it can be qualified with "as of now" in case this is missed during future updates.
```suggestion
CUDA 13 this ranges from 7.5 to 12.1 (the latest as of CUDA 13.0), depending on the exact installed version.
```